### ISSUE TITLE
Resolution of blender dataset -- reproducing issue

### ISSUE THREAD

**Issue by wsunid** [sjtuzq/point-radiance#3]
Hi,

At first, thanks for sharing the code! 

I'm wondering about the resolution blender dataset you used in table 1?  I run your code with the given cmd and realize the resolution is 400x400 whereas the resolution of other baselines use the resolution of 800x800. 

Not sure if I mess something up. After I switch into the resolution of 800x800 by setting `half_res = False` and `img_s = 800`, I found the FID score for chair was decreased a lot (27 for chair vs 32.98 reported in the supplementary).

All the best,
Weiwei.

**Comment by wsunid** [comment]
For the convenience of you to help debugging, I describe how I get these numbers:

I was using the standard test split by tweaking the ids here https://github.com/sjtuzq/point-radiance/blob/501158f58b70ea38b11f06715be8435cb09f8df8/main.py#L151 -->`psnr_e = trainer.test(200, 400, True)`. 

For chair, at the resolution of 400x400, I got the number of 29.60 with 3.55 mins whereas the reported number is 32.98.  At the resolution of 800x800, I got the number of 26.91 with 10 mins.

Basically, I'm having problem reproducing the number reported in the paper. I would appreciate much if you can provide any suggestions! 

Thanks in advance!

**Comment by zvict** [comment]
I face the same issue when I try to compare your work as a baseline with more metrics.
Also it would be very helpful if you could share the configs for the reported scores for all the scenes.

Thanks so much!

**Comment by sjtuzq** [comment]
Hi, Please see lines 130 to line 132 in modules/config.py for the specified parameters for all the scenes.

**Comment by wsunid** [comment]
Thanks for the reply! 

To make sure I understand correctly, did you use the blender dataset with the lower resolution of 400?

Thanks, 
Weiwei.

**Comment by sjtuzq** [comment]
Yep, the default resolution is 400.

**Comment by wsunid** [comment]
Thanks for letting me know!
